packages/@lwc/perf-benchmarks/scripts/build.js

```diff
@@ -12,7 +12,7 @@
 const path = require('path');
 const fs = require('fs');
 const { promisify } = require('util');
-const glob = promisify(require('glob'));
+const { glob } = require('glob');
```

Comment:
@nolanlawson I wasn't able to use the 

```javascript
const { glob } = promisify(require('glob'));
```

syntax as the return value of `promisify` is a promise, but looking at the function signature for `glob`, it looks like it already returns a promise.

I import the `glob` function directly and it seems to be working, I don't think it should cause any issues but wanted to get your thoughts.

Alternatively we could do something like this

```javascript
const glob = promisify(require('glob').glob);
```

Reply:
@jmsjtu I just tested, and actually you were right; it does return a promise. `require('glob').glob` is an `AsyncFunction`. So you should be able to just do `const { glob } = require('glob')` and skip the promisify.

Reply:
Like, when I run this, it works fine:

```js
require('glob').glob('/tmp/*').then(console.log)
```